scripts/ramp_build_demand_profile.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import logging
+import os
+from pathlib import Path
+
+import pandas as pd
+import yaml
+from _helpers_dist import configure_logging, create_logger, read_osm_config
+from ramp import Appliance, UseCase, User, get_day_type
+
+
+def create_demand_profile(xlsx_input_path, excel_output_path):
+    use_case = UseCase()
+    use_case.load(xlsx_input_path)
+    # Versione che fa solo una simulazione:
+    n_days = 365
```

Comment:
this one could be an input to the function and loaded from the config file; we can keep this for the next PR